src/graphs.py

```diff
@@ -21,13 +23,16 @@ def open_files(self, files, import_settings):
                 import_settings.path = path
                 xdata, ydata = file_io.get_data(self, import_settings)
                 if xdata == []:
-                    filename = import_settings.path.split("/")[-1]
+                    filename = pathlib.Path(import_settings.path).stem
```

Comment:
Not 100% what stem does, will check how functional it is but if it works I'm fine with this as I'm not 100% sold on using splits either.